### PR TITLE
[zh-cn]Sync disruptions.md

### PR DESCRIPTION
**Pull request by yanrongshi**
Synchronized some upstream changes and rewrote content and format.
```yaml
content/zh-cn/docs/concepts/workloads/pods/disruptions.md
```

### CODE REVIEW

File: content/zh-cn/docs/concepts/workloads/pods/disruptions.md [hunk 1]
```diff
@@ -179,17 +179,17 @@ time as frequent voluntary disruptions.  We call this set of features
...
 <!--
 ## Pod disruption budgets
 
 Kubernetes offers features to help you run highly available applications even when you
 introduce frequent voluntary disruptions.
 
-An Application Owner can create a `PodDisruptionBudget` object (PDB) for each application.
-A PDB limits the number of pods of a replicated application that are down simultaneously from
-voluntary disruptions.  For example, a quorum-based application would
+As an application owner, you can create a PodDisruptionBudget (PDB) for each application.
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/pods/disruptions.md:190]: the zh-cn text shall be updated too.

File: content/zh-cn/docs/concepts/workloads/pods/disruptions.md [hunk 2]
```diff
@@ -219,24 +218,25 @@ and the Kubernetes-on-GCE cluster upgrade script (`cluster/gce/upgrade.sh`).
...
-are terminated, or until a configurable timeout is reached.
+the Node you're taking out of service. The eviction request that `kubectl` submits on
+your behalf may be temporarily rejected, so the tool periodically retries all failed
+requests until all Pods on the target node are terminated, or until a configurable timeout is reached.
 -->
 例如，`kubectl drain` 命令可以用来标记某个节点即将停止服务。
 运行 `kubectl drain` 命令时，工具会尝试驱逐机器上的所有 Pod。
-`kubectl` 所提交的驱逐请求可能会暂时被拒绝，所以该工具会定时重试失败的请求，
-直到所有的 Pod 都被终止，或者达到配置的超时时间。
+`kubectl` 所提交的驱逐请求可能会暂时被拒绝，
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/pods/disruptions.md:227]:
```suggestion
`kubectl` 代表你所提交的驱逐请求可能会暂时被拒绝，
```

File: content/zh-cn/docs/concepts/workloads/pods/disruptions.md [hunk 3]
```diff
@@ -219,24 +218,25 @@ and the Kubernetes-on-GCE cluster upgrade script (`cluster/gce/upgrade.sh`).
...
+所以该工具会定时重试失败的请求，
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/pods/disruptions.md:228]:
```suggestion
所以该工具会定时重试所有失败的请求，
```

File: content/zh-cn/docs/concepts/workloads/pods/disruptions.md [hunk 4]
```diff
@@ -282,14 +285,13 @@ hornoring the
...
-## PDB Example
-
+## PodDisruptionBudget example {#pdb-example}
 Consider a cluster with 3 nodes, `node-1` through `node-3`.
 The cluster is running several applications.  One of them has 3 replicas initially called
 `pod-a`, `pod-b`, and `pod-c`.  Another, unrelated pod without a PDB, called `pod-x`, is also shown.
 Initially, the pods are laid out as follows:
 -->
-## PDB 例子   {#pdb-example}
+## PodDisruptionBudget（PDB） 例子   {#pdb-example}
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/pods/disruptions.md:294]:
```suggestion
## PodDisruptionBudget 例子   {#pdb-example}
```
As a heading or title, it is few to use the form of `PodDisruptionBudget (PDB)`, which often appears in the begining of body.

File: content/zh-cn/docs/concepts/workloads/pods/disruptions.md [hunk 5]
```diff
@@ -199,18 +199,17 @@ percentage of the total.
 
 {{< feature-state for_k8s_version="v1.21" state="stable" >}}
 
-即使你会经常引入自愿性干扰，Kubernetes 也能够支持你运行高度可用的应用。
+即使你会经常引入自愿性干扰，Kubernetes 提供的功能也能够支持你运行高度可用的应用。
 
-应用程序所有者可以为每个应用程序创建 `PodDisruptionBudget` 对象（PDB）。
+作为一个应用程序的所有者，你可以为每个应用程序创建一个 `PodDisruptionBudget` 对象（PDB）。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/pods/disruptions.md:204]:
```suggestion
作为一个应用程序的所有者，你可以为每个应用程序创建一个 PodDisruptionBudget（PDB）。
```

File: content/zh-cn/docs/concepts/workloads/pods/disruptions.md [hunk 6]
```diff
@@ -199,18 +199,17 @@ percentage of the total.
...
 PDB 将限制在同一时间因自愿干扰导致的复制应用程序中宕机的 pod 数量。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/pods/disruptions.md:205]:
```suggestion
PDB 将限制在同一时间因自愿干扰导致的复制应用程序中宕机的 Pod 数量。
```

File: content/zh-cn/docs/concepts/workloads/pods/disruptions.md [hunk 7]
```diff
@@ -262,13 +264,14 @@ PDB 无法防止[非自愿干扰](#voluntary-and-involuntary-disruptions)；
...
-are not limited by PDBs when doing rolling upgrades - the handling of failures
-during application updates is configured in spec for the specific workload resource.
+against the disruption budget, but workload resources (such as Deployment and StatefulSet)
+are not limited by PDBs when doing rolling upgrades. Instead, the handling of failures
+during application updates is configured in the spec for the specific workload resource.
 -->
 由于应用程序的滚动升级而被删除或不可用的 Pod 确实会计入干扰预算，
-但是控制器（如 Deployment 和 StatefulSet）在进行滚动升级时不受 PDB
-的限制。应用程序更新期间的故障处理方式是在对应的工作负载资源的 `spec` 中配置的。
+但是工作复杂资源（如 Deployment 和 StatefulSet）
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/pods/disruptions.md:272]:
```suggestion
但是工作负载资源（如 Deployment 和 StatefulSet）
```
Not necessary to add newline here.

File: content/zh-cn/docs/concepts/workloads/pods/disruptions.md [hunk 8]
```diff
@@ -262,13 +264,14 @@ PDB 无法防止[非自愿干扰](#voluntary-and-involuntary-disruptions)；
...
+在进行滚动升级时不受 PDB的限制。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/pods/disruptions.md:273]:
```suggestion
在进行滚动升级时不受这些 PDB 的限制。
```

File: content/zh-cn/docs/concepts/workloads/pods/disruptions.md [hunk 9]
```diff
@@ -199,18 +199,17 @@ percentage of the total.
 
 {{< feature-state for_k8s_version="v1.21" state="stable" >}}
 
-即使你会经常引入自愿性干扰，Kubernetes 也能够支持你运行高度可用的应用。
+即使你会经常引入自愿性干扰，Kubernetes 提供的功能也能够支持你运行高度可用的应用。
 
-应用程序所有者可以为每个应用程序创建 `PodDisruptionBudget` 对象（PDB）。
-PDB 将限制在同一时间因自愿干扰导致的复制应用程序中宕机的 pod 数量。
-例如，基于票选机制的应用程序希望确保运行的副本数永远不会低于仲裁所需的数量。
+作为一个应用程序的所有者，你可以为每个应用程序创建一个 `PodDisruptionBudget`（PDB）。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/pods/disruptions.md:204]:
```suggestion
作为一个应用程序的所有者，你可以为每个应用创建一个 `PodDisruptionBudget`（PDB）。
```

File: content/zh-cn/docs/concepts/workloads/pods/disruptions.md [hunk 10]
```diff
@@ -199,18 +199,17 @@ percentage of the total.
...
+PDB 将限制在同一时间因自愿干扰导致的复制应用程序中宕机的 Pod 数量。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/pods/disruptions.md:205]:
```suggestion
PDB 将限制在同一时间因自愿干扰导致的多副本应用中发生宕机的 Pod 数量。
```

File: content/zh-cn/docs/concepts/workloads/pods/disruptions.md [hunk 11]
```diff
@@ -199,18 +199,17 @@ percentage of the total.
...
+例如，基于 Quorum 机制的应用程序希望确保运行的副本数永远不会低于 Quorum 所需的数量。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/pods/disruptions.md:206]:
```suggestion
例如，基于票选机制的应用希望确保运行中的副本数永远不会低于票选所需的数量。
```

File: content/zh-cn/docs/concepts/workloads/pods/disruptions.md [hunk 12]
```diff
@@ -219,24 +218,25 @@ and the Kubernetes-on-GCE cluster upgrade script (`cluster/gce/upgrade.sh`).
...
 service. When you run `kubectl drain`, the tool tries to evict all of the Pods on
-the Node you'are taking out of service. The eviction request may be temporarily rejected,
-and the tool periodically retries all failed requests until all pods
-are terminated, or until a configurable timeout is reached.
+the Node you're taking out of service. The eviction request that `kubectl` submits on
+your behalf may be temporarily rejected, so the tool periodically retries all failed
+requests until all Pods on the target node are terminated, or until a configurable timeout is reached.
 -->
 例如，`kubectl drain` 命令可以用来标记某个节点即将停止服务。
 运行 `kubectl drain` 命令时，工具会尝试驱逐机器上的所有 Pod。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/pods/disruptions.md:226]:
```suggestion
运行 `kubectl drain` 命令时，工具会尝试驱逐你所停服的节点上的所有 Pod。
```

File: content/zh-cn/docs/concepts/workloads/pods/disruptions.md [hunk 13]
```diff
@@ -219,24 +218,25 @@ and the Kubernetes-on-GCE cluster upgrade script (`cluster/gce/upgrade.sh`).
...
-`kubectl` 所提交的驱逐请求可能会暂时被拒绝，所以该工具会定时重试失败的请求，
-直到所有的 Pod 都被终止，或者达到配置的超时时间。
+`kubectl` 代表你所提交的驱逐请求可能会暂时被拒绝，
+所以该工具会定时重试所有失败的请求，
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/pods/disruptions.md:228]:
```suggestion
所以该工具会周期性地重试所有失败的请求，
```

File: content/zh-cn/docs/concepts/workloads/pods/disruptions.md [hunk 14]
```diff
@@ -262,13 +264,14 @@ PDB 无法防止[非自愿干扰](#voluntary-and-involuntary-disruptions)；
...
+against the disruption budget, but workload resources (such as Deployment and StatefulSet)
+are not limited by PDBs when doing rolling upgrades. Instead, the handling of failures
+during application updates is configured in the spec for the specific workload resource.
 -->
 由于应用程序的滚动升级而被删除或不可用的 Pod 确实会计入干扰预算，
-但是控制器（如 Deployment 和 StatefulSet）在进行滚动升级时不受 PDB
-的限制。应用程序更新期间的故障处理方式是在对应的工作负载资源的 `spec` 中配置的。
+但是工作负载资源（如 Deployment 和 StatefulSet）
+在进行滚动升级时不受 PDB 的限制。
+应用程序更新期间的故障处理方式是在对应的工作负载资源的 `spec` 中配置的。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/pods/disruptions.md:274]:
```suggestion
应用更新期间的故障处理方式是在对应的工作负载资源的 `spec` 中配置的。
```

**Review by Sea-n — COMMENTED** [review]
/lgtm

**Pull request merged**